system_tests/full_challenge_impl_test.go

```diff
@@ -206,6 +206,13 @@ func setupSequencerInboxStub(ctx context.Context, t *testing.T, l1Info *Blockcha
 		DelaySeconds:  big.NewInt(10000),
 		FutureSeconds: big.NewInt(10000),
 	}
+
+	//  Deploy EspressoTEEVerifier Mock
+	espressoTEEVerifierAddr, tx, _, err := mocksgen.DeployEspressoTEEVerifierMock(&txOpts, l1Client)
```

Comment:
Should this deployment be written in `deploy.go`?

Reply:
This is a test file, I think this should be fine.